extension/content.js

```diff
@@ -99,10 +99,26 @@ function renderVoteCount(type, count) {
 		iconUrl = 'https://assets-cdn.github.com/images/icons/emoji/unicode/1f44e.png';
 	}
 	const $sidebar = $('#partial-discussion-sidebar');
+	let avatars = '';
+
+	voters.forEach(val => {
+		avatars += val;
+	});
+
+	avatars = avatars.replace(/height="48"/g, 'height="20"')
+										.replace(/width="48"/g, 'width="20"')
+										.replace(/<a href/g, '<a class="participant-avatar" href')
+										.replace(/timeline-comment-avatar/g, 'avatar');
```

Comment:
This is the solution I came up with using a detached node, but I'm not crazy about it. If you think it's better I'll commit and push.

```js
	const avatarBlock = $('<div>').append(avatars);
	$(avatarBlock).children('a').map(function () {
		$(this).addClass('participant-avatar');
		$($(this)).children('img').map(function () {
			$(this).attr('height', 20).attr('width', 20).attr('class', 'avatar');
			return this;
		});
		return this;
	});
```

The reason I'm doing `return this` is because without it I get this error from XO:

```
error  Expected to return a value in this function  array-callback-return
```